modules/integration/src/test/java/org/apache/fluo/integration/TestUtil.java

```diff
@@ -17,18 +17,26 @@
 
 import org.apache.fluo.api.client.SnapshotBase;
 import org.apache.fluo.api.client.TransactionBase;
+import org.apache.fluo.api.data.Bytes;
 import org.apache.fluo.api.data.Column;
 
 public class TestUtil {
 
   private TestUtil() {}
 
+  private static final Bytes ZERO = Bytes.of("0");
+
+  public static void increment(TransactionBase tx, Bytes row, Column col, int val) {
+    int prev = 0;
+    String prevStr = tx.get(row, col, ZERO).toString();
```

Comment:
I think its safe since its internal test code and we have full control over all of the code that calls it.